app/controllers/application_controller.rb

```diff
@@ -11,5 +11,12 @@ class ApplicationController < ActionController::Base
   def current_user
     @current_user ||= User.where(id: session[:user_id]).first if session[:user_id]
   end
-  helper_method :current_user
+
+  def remove_cookie_banner(s)
+    #  Hot patch to remove the cookie banner before rendering
+    cookie_banner_regex = %r{<div id="global-cookie-message">\s*My Cookie Message\s*<\/div>}
```

Comment:
Can we leverage the fact this is a regex and use a wildcard instead of `My Cookie Message`? This means that should this ever fail (for reasons we're sadly unaware of) the default copy is still displayed and is probably more informative for our users.

Reply:
The wildcard wasn't working for me yesterday. Figured out today that there were whitespaces preventing it, so changed back

Reply:
An alternate solution: 

```diff
diff --git a/app/controllers/application_controller.rb b/app/controllers/application_controller.rb
index 85e2e38..c8cb036 100644
--- a/app/controllers/application_controller.rb
+++ b/app/controllers/application_controller.rb
@@ -13,9 +13,10 @@ private
   end
 
   def remove_cookie_banner(content)
-    #  Hot patch to remove the cookie banner before rendering
-    cookie_banner_regex = %r{<div id="global-cookie-message">\s*My Cookie Message\s*<\/div>}
-    content.gsub(cookie_banner_regex, '')
+    doc = Nokogiri::HTML.fragment(content)
+    doc.search('#global-cookie-message').remove
+
+    doc.to_html
   end
 
   helper_method :current_user, :remove_cookie_banner
```